tracecontext.go

```diff
@@ -204,6 +205,58 @@ func NewTraceState(entries ...TraceStateEntry) TraceState {
 	return out
 }
 
+// ParseTraceState attempts to decode a TraceState from the passed
+// string. It returns an error if the input is invalid according to the W3C
+// Trace Context specification.
+// Adapted from https://github.com/open-telemetry/opentelemetry-go/blob/trace/v1.3.0/trace/tracestate.go.
```

Comment:
We have our own parse function already: `apmhttp.ParseTracestateHeader`. Does this do something we don't already do? If we need it elsewhere, should we just move it to the top-level package?

Reply:
ahh, I only looked for this in the top-level package. nope, looks like it does the thing!